frontend/views/HomeView.tsx

```diff
@@ -0,0 +1,51 @@
+import Link from "next/link";
+import { Button } from "@/components/ui/button";
+
+export default function HomeView() {
+  return (
+    <div
+      className="flex flex-col w-full justify-left p-40 bg-secondary"
+      style={{
+        backgroundImage: 'url("course_map.jpg")',
+        backgroundSize: "75%", // Set a specific size, e.g., 30% of the container
+        backgroundPosition: "right center", // Align to the left, centered vertically
+        backgroundRepeat: "no-repeat", // Crucial: prevents the image from tiling
+      }}
+    >
```

Comment:
I think we can avoid using inline style and stick to tailwind. Also, there is some optimization done by nextJS. You could do something like

```suggestion
      <div className="relative flex flex-col w-full justify-start p-40 bg-secondary">
  <Image
    src="/course_map.jpg"
    alt="Course map"
    fill
    className="object-[75%_center] object-contain"
  />
  <div className="relative z-10">
    {/* content here */}
  </div>
</div>
```

Reply:
Det ger en liten omständigare lösning, men kan fixa för optimiseringsskäl. Vet dock inte hur stor roll det spelar i slutändan